CorsixTH/Lua/room.lua

```diff
@@ -808,13 +808,14 @@ function Room:crashRoom()
     self.door.reserved_for = nil
   end
 
-  local remove_humanoid = function(humanoid)
+  local remove_humanoid = function(humanoid, room)
```

Comment:
This function already knows about room (self) (see line 822). Not needed but preference would be to rewrite the top line as `local function removeHumanoid(humanoid) ...` but either method is correct.